README.md

```diff
@@ -86,7 +86,14 @@ CASA participants are structured into three groups:
 - Boris Mann @bmann (Fission Founder),
 - Chairman/Tiebreaker: Juan Caballero @bumblefudge (Acting Executive Director, Independent)
 
-The Board members above directly control a [multi-signature "vault"](https://app.safe.global/home?safe=eth:0xEDA75C4e519FD27d7d28B7F20be407435fb74e2d) for transparent, on-chain grant reporting and other disbursements.
+The Board members above directly control a [multi-signature "vault"](https://app.safe.global/home?safe=eth:0xEDA75C4e519FD27d7d28B7F20be407435fb74e2d) for transparent, on-chain grant reporting and other disbursements. Respectively, the addresses with which they control the safe are:
+- eip155:1:0xcBec15583A21C3ddAD5fab658be5b4FE85Df730b
```

Comment:
The addresses should be near the names of the board members, not in a separate paragraph